api/tools/src/main/java/org/pmiops/workbench/tools/cdrconfig/UpdateCdrConfig.java

```diff
@@ -241,7 +241,7 @@ private void updateDB(
       currentCdrVersions.put(cdrVersion.getCdrVersionId(), cdrVersion);
     }
 
-    List<DbCdrVersion> cdrVersions = cdrConfigMapper.cdrVersions(cdrConfig);
+    List<DbCdrVersion> cdrVersions = cdrConfigMapper.cdrVersions(cdrConfig, accessTierDao);
```

Comment:
To clarify - the issue I saw is that there were duplicate elements in the `accessTiers` array, i.e. L194. Is that consistent with the fix here? I can't really work backwards to this, but it seems the cause was some odd interaction between the mapper and dao, so it's possible this fixes it.

To verify, when you run dev-up, logs should indicate that `accessTiers` are unchanged (unless it's your first time running dev-up).

Reply:
yes - I do see accessTiers are unchanged on dev-up

Reply:
The accessTiers array is populated via `mapper.toDbTiers()` which is where the problem occurred.  So this does make sense.